### DISCUSSION TITLE
createSafeActionClient() returns possibly undefined in v7?

### DISCUSSION THREAD

**Discussion by chris-orgorg** [next-safe-action/next-safe-action#211] [Q&A]
Hi, I've upgraded from v5 to v7 and noticed that now in v7, the action instance that createSafeActionClient() returns can be undefined.  What's the reason for this?  It seems client code would be cleaner without having to check for undefined all the time.

**Comment by TheEdoRan** [comment]
See:
- https://github.com/TheEdoRan/next-safe-action/discussions/175
